pkg/analysis/passes/gomanifest/gomanifest.go

```diff
@@ -167,43 +176,23 @@ func verifyManifest(manifest map[string]string, goFiles []string, sourceCodeDir
 	return nil
 }
 
-func hashFileContent(path string) (string, error) {
-	// Handle hashing big files.
-	// Source: https://stackoverflow.com/q/60328216/1722542
-
-	f, err := os.Open(path)
+// we hash the file content using the sha256 algorithm
+// a second hash is created using the line endings that windows uses
+// for plugins compiled with native windows that use windows line endings
+func hashFileContent(path string) (string, string, error) {
```

Comment:
because I now have to essentially run replace in the whole content I went with reading the whole file instead of chunks. It is very unlikely we'll have go files big enough to overflow the memory